### PR TITLE
[18.0][MIG] fieldservice_recurring

### PR DESCRIPTION
**Pull request by EdgarRetes**
@max3903

### CODE REVIEW

**Comment by max3903** [comment]
@EdgarRetes Can you fix the pre-commit errors?

**Comment by EdgarRetes** [comment]
@brian10048 @max3903 can you check this?

File: fieldservice_recurring/static/description/icon.png
Comment (max3903) [inline fieldservice_recurring/static/description/icon.png:1]: @EdgarRetes Can you update the icon with the one form `fieldservice` please?

File: fieldservice_recurring/models/fsm_recurring.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    fsm_frequency_set_id = fields.Many2one(
+        "fsm.frequency.set",
+        "Frequency Set",
+    )
+    scheduled_duration = fields.Float(help="Scheduled duration of the work in hours")
+
+    start_date = fields.Datetime()
+    end_date = fields.Datetime(help="Recurring orders will not be made after this date")
+    max_orders = fields.Integer(
+        string="Maximum Orders", help="Maximium number of orders that will be created"
```
Comment (max3903) [inline fieldservice_recurring/models/fsm_recurring.py]:
```suggestion
        string="Maximum Orders", help="Maximum number of orders that will be created"
```

**Review by max3903 — CHANGES_REQUESTED** [review]
Small fixes. :+1:

**Review by brian10048 — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Pull request merged**